### DISCUSSION TITLE
convert data from CP2K aimd output

### DISCUSSION THREAD

**Discussion by WeJear** [deepmodeling/dpdata#578] [Q&A]
When I was using the dpdata package to convert the output of cp2k aimd, some errors occurred. Hopefully someone can help me figure out what the problem is.

> import dpdata
> data = dpdata.LabeledSystem('./water-dp', cp2k_output_name = 'md.log', fmt="cp2kdata/md")
> print(data)

The following is the error：

> --- You are parsing data using package Cp2kData ---
> Traceback (most recent call last):
> File "/home/xwj/materials/water/water-deepmd/water-dp.py", line 10, in
> data = dpdata.LabeledSystem('./{}-dp'.format(file_label), cp2k_output_name = 'md.log', fmt="cp2kdata/md")
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/dpdata/system.py", line 183, in init
> self.from_fmt(
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/dpdata/system.py", line 220, in from_fmt
> return self.from_fmt_obj(load_format(fmt), file_name, **kwargs)
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/dpdata/system.py", line 1113, in from_fmt_obj
> data = fmtobj.from_labeled_system(file_name, **kwargs)
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/cp2kdata/dpdata_plugin.py", line 73, in from_labeled_system
> cp2kmd = Cp2kOutput(output_file=cp2k_output_name, run_type="MD", path_prefix=path_prefix)
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/cp2kdata/output.py", line 68, in init
> self.dft_info = parse_dft_info(self.filename)
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/cp2kdata/block_parser/header_info.py", line 91, in parse_dft_info
> return DFTInfo(ks_type=dft_info["ks_type"][0][0][0], multiplicity=dft_info["multiplicity"][0][0][0])
> IndexError: list index out of range
> (deepmd) xwj@ERNIC:~/materials/water/water-deepmd$ python water-dp.py
> --- You are parsing data using package Cp2kData ---
> Traceback (most recent call last):
> File "/home/xwj/materials/water/water-deepmd/water-dp.py", line 10, in
> data = dpdata.LabeledSystem('./{}-dp'.format(file_label), cp2k_output_name = 'md.log', fmt="cp2kdata/md")
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/dpdata/system.py", line 183, in init
> self.from_fmt(
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/dpdata/system.py", line 220, in from_fmt
> return self.from_fmt_obj(load_format(fmt), file_name, **kwargs)
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/dpdata/system.py", line 1113, in from_fmt_obj
> data = fmtobj.from_labeled_system(file_name, **kwargs)
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/cp2kdata/dpdata_plugin.py", line 73, in from_labeled_system
> cp2kmd = Cp2kOutput(output_file=cp2k_output_name, run_type="MD", path_prefix=path_prefix)
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/cp2kdata/output.py", line 68, in init
> self.dft_info = parse_dft_info(self.filename)
> File "/home/xwj/miniconda3/envs/deepmd/lib/python3.10/site-packages/cp2kdata/block_parser/header_info.py", line 91, in parse_dft_info
> return DFTInfo(ks_type=dft_info["ks_type"][0][0][0], multiplicity=dft_info["multiplicity"][0][0][0])
> IndexError: list index out of range

I provided my .inp file, the other files are too big for me to upload：
[water-dp.zip](https://github.com/deepmodeling/dpdata/files/13345413/water-dp.zip)

**Comment by robinzyb** [comment]
transfer to here. https://github.com/robinzyb/cp2kdata/issues/26
